src/test/java/ProfileTests.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+import pages.ProfilePage;
+
+import java.util.UUID;
+
+public class ProfileTests extends BaseTest{
+
+    @Test
+    public void changeProfileName() throws InterruptedException {
+
+        navigateToProfilePage();
+        String uniqueName = generateUniqueName();
+        Thread.sleep(2000);
+        changeName(uniqueName);
+        Thread.sleep(2000);
+    }
```

Comment:
**Use in test methods, the following logical split   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/8cbc9b96-fbc2-49de-abcb-29335cfaf5b9)